src/transport.js

```diff
@@ -67,14 +68,19 @@ module.exports = function (swarm) {
               log('dial canceled: %s', multiaddr.toString())
               // clean up already done dials
               if (conn) {
-                conn.close()
+                pull(
+                  pull.empty(),
+                  conn
+                )
+
+                // conn.close()
               }
               return cb()
             }
 
             // one is enough
             log('dial success: %s', multiaddr.toString())
-            q.kill()
+            q.kill() // why kill here?
```

Comment:
yeah, fixed the error  above https://github.com/libp2p/js-libp2p-swarm/pull/202#issue-217352527 by not reusing queues